ConfigurationSystem/private/Modificator.py

```diff
@@ -77,202 +78,202 @@ def getDictRootedAt(path):
+        cfg = cfg[section]
       return sectionList[-1] in cfg.listOptions()
-    except:
+    except Exception:
       return False
 
-  def renameKey( self, path, newName ):
-    parentCfg = self.cfgData.getRecursive( path, -1 )
+  def renameKey(self, path, newName):
+    parentCfg = self.cfgData.getRecursive(path, -1)
     if not parentCfg:
       return False
-    pathList = List.fromChar( path, "/" )
+    pathList = List.fromChar(path, "/")
     oldName = pathList[-1]
-    if parentCfg[ 'value' ].renameKey( oldName, newName ):
+    if parentCfg['value'].renameKey(oldName, newName):
       pathList[-1] = newName
-      self.__setCommiter( "/%s" % "/".join( pathList ) )
+      self.__setCommiter("/%s" % "/".join(pathList))
       return True
     else:
       return False
 
-  def copyKey( self, originalKeyPath, newKey ):
-    parentCfg = self.cfgData.getRecursive( originalKeyPath, -1 )
+  def copyKey(self, originalKeyPath, newKey):
+    parentCfg = self.cfgData.getRecursive(originalKeyPath, -1)
     if not parentCfg:
       return False
-    pathList = List.fromChar( originalKeyPath, "/" )
+    pathList = List.fromChar(originalKeyPath, "/")
     originalKey = pathList[-1]
-    if parentCfg[ 'value' ].copyKey( originalKey, newKey ):
-      self.__setCommiter( "/%s/%s" % ( "/".join( pathList[:-1] ), newKey ) )
+    if parentCfg['value'].copyKey(originalKey, newKey):
+      self.__setCommiter("/%s/%s" % ("/".join(pathList[:-1]), newKey))
       return True
     return False
 
-  def removeOption( self, optionPath ):
-    if not self.existsOption( optionPath ):
+  def removeOption(self, optionPath):
+    if not self.existsOption(optionPath):
       return False
-    cfg = self.__getParentCFG( optionPath )
-    optionName = List.fromChar( optionPath, "/" )[-1]
-    return cfg.deleteKey( optionName )
+    cfg = self.__getParentCFG(optionPath)
+    optionName = List.fromChar(optionPath, "/")[-1]
+    return cfg.deleteKey(optionName)
 
-  def removeSection( self, sectionPath ):
-    if not self.existsSection( sectionPath ):
+  def removeSection(self, sectionPath):
+    if not self.existsSection(sectionPath):
       return False
-    cfg = self.__getParentCFG( sectionPath )
-    sectionName = List.fromChar( sectionPath, "/" )[-1]
-    return cfg.deleteKey( sectionName )
+    cfg = self.__getParentCFG(sectionPath)
+    sectionName = List.fromChar(sectionPath, "/")[-1]
+    return cfg.deleteKey(sectionName)
 
-  def loadFromBuffer( self, data ):
+  def loadFromBuffer(self, data):
     self.cfgData = CFG()
-    self.cfgData.loadFromBuffer( data )
+    self.cfgData.loadFromBuffer(data)
 
-  def loadFromFile( self, filename ):
+  def loadFromFile(self, filename):
     self.cfgData = CFG()
-    self.mergeFromFile( filename )
+    self.mergeFromFile(filename)
 
-  def dumpToFile( self, filename ):
-    fd = file( filename, "w" )
-    fd.write( str( self.cfgData ) )
+  def dumpToFile(self, filename):
+    fd = open(filename, "w")
```

Comment:
```suggestion
    with open(filename, "wt") as fd:
        fd.write(str(self.cfgData))
```
(`t` is the default mode, but I always prefer to be explicit)